libp2p/Host.h

```diff
@@ -266,6 +266,11 @@ class Host: public Worker
 
     std::shared_ptr<CapabilityHostFace> capabilityHost() const { return m_capabilityHost; }
 
+    /// Execute work on the network thread after an @_expiryDelay delay.
```

Comment:
That was doxygen syntax to refer to the parameter inside description
(http://www.doxygen.nl/manual/commands.html#cmda)

```suggestion
    /// Execute work on the network thread after an @a _expiryDelay delay.
```

Reply:
@gumb0 That's what I suspected but I saw different syntax used in other parts of the project e.g.: https://github.com/ethereum/aleth/blob/78355a28da55f1c0e952e3cf1a17891cea981cb4/libethereum/Account.h#L150-L152